src/Compatibility/Core/src/Windows/FontImageSourceHandler.cs

```diff
@@ -17,6 +18,10 @@ public sealed class FontImageSourceHandler : IImageSourceHandler, IIconElementHa
 	{
 		float _minimumDpi = 300;
 
+		IFontManager _fontManager;
+		IFontManager FontManager
+			=> _fontManager ??= Application.Current?.FindMauiContext()?.Services?.GetService<IFontManager>();
```

Comment:
You could use `imagesource` as the starting point to search for the MauiContext 

`imagesource.FindMauiContext`